src/bin/barracks

```diff
@@ -13,7 +13,8 @@ if (config.v2Enabled) {
     .command('device [cmd]', 'Manage devices')
     .command('filter [cmd]', 'Manage filters')
     .command('package [cmd]', 'Manage packages')
-    .command('token [cmd]', 'Manage tokens');
+    .command('token [cmd]', 'Manage tokens')
+    .command('integration [cmd]', 'Manage third-party integrations');
```

Comment:
It is still considered as experimental in V2, please add it in a config.experimental condition